--- origsrc/libgsf-1.14.9/gsf/Makefile.am	2008-03-17 04:15:30.000000000 -0500
+++ src/libgsf-1.14.9/gsf/Makefile.am	2008-09-03 12:12:13.287011200 -0500
@@ -134,7 +134,7 @@
 
 include $(top_srcdir)/gsf.mk
 CPP_CFLAGS = -DGLIB_CHECK_VERSION\(x,y,z\)
-LIB_PUBLIC_HDRS = $(libgsf_1_include_HEADERS)
+LIB_PUBLIC_HDRS = $(libgsf_1_include_HEADERS) gsf-shared-memory.h
 
 if HAVE_LIBEXE
 gsf_mslibdir = $(DESTDIR)$(libdir)
