--- origsrc/libgsf-1.14.20/gsf/Makefile.am
+++ src/libgsf-1.14.20/gsf/Makefile.am
@@ -34,7 +34,7 @@ IGNORE_HFILES=			 \
 	gsf-output-transaction.h \
 	gsf-infile-ar.h
 
-EXTRA_HFILES = ../gsf-gnome/*.h
+EXTRA_HFILES = $(top_srcdir)/gsf-gnome/*.h
 
 # Images to copy into HTML directory
 HTML_IMAGES =
@@ -134,7 +134,7 @@
 
 include $(top_srcdir)/gsf.mk
 CPP_CFLAGS = -DGLIB_CHECK_VERSION\(x,y,z\)
-LIB_PUBLIC_HDRS = $(libgsf_1_include_HEADERS)
+LIB_PUBLIC_HDRS = $(libgsf_1_include_HEADERS) gsf-shared-memory.h
 
 if HAVE_LIBEXE
 gsf_mslibdir = $(DESTDIR)$(libdir)
